src/haxelib/client/ConvertXml.hx

```diff
@@ -58,9 +58,14 @@ class ConvertXml {
 						case "description":
 							json.description = node.firstChild().toString();
 						case "depends":
-							var name = node.get("name");
-							var version = node.get("version");
-							if (version == null) version = "";
+							final name = node.get("name");
+							final version = {
```

Comment:
"With a final in hand, everything looks like a countdown"

Frankly, for cases like this I prefer sticking to `var` instead of making awkward workarounds just because...

Reply:
Hm, is it fine to test with haxe nightly so we can use the null coalescing operator here?